site/src/api/errors.test.ts

```diff
@@ -36,3 +36,57 @@ describe("mapApiErrorToFieldErrors", () => {
     })
   })
 })
+
+describe("getValidationErrorMessage", () => {
+  it("returns multiple validation messages", () => {
+    expect(
+      getValidationErrorMessage({
+        response: {
+          data: {
+            message: "Invalid user search query.",
+            validations: [
+              {
+                field: "status",
+                detail: `Query param "status" has invalid value: "inactive" is not a valid user status`,
+              },
+              {
+                field: "q",
+                detail: `Query element "role:a:e" can only contain 1 ':'`,
+              },
+            ],
+          },
+        },
+        isAxiosError: true,
+      }),
+    ).toEqual(
+      `Query param "status" has invalid value: "inactive" is not a valid user status\nQuery element "role:a:e" can only contain 1 ':'`,
+    )
+  })
+
+  it("non-API error returns empty validation message", () => {
+    expect(
+      getValidationErrorMessage({
+        response: {
+          data: {
+            error: "Invalid user search query.",
+          },
+        },
+        isAxiosError: true,
+      }),
+    ).toEqual("")
+  })
+
+  it("no validations field returns empty validation message", () => {
+    expect(
+      getValidationErrorMessage({
+        response: {
+          data: {
+            message: "Invalid user search query.",
+            detail: `Query element "role:a:e" can only contain 1 ':'`,
+          },
+        },
+        isAxiosError: true,
+      }),
+    ).toEqual("")
```

Comment:
What value are these last two test cases providing? Is the idea that we are testing the shape of the API response?

Reply:
Yes! In case it is not a validation error, but a different error or a different API error, the method should gracefully handle the case.